paymentsheet/src/test/java/com/stripe/android/link/ui/wallet/WalletScreenTest.kt

```diff
@@ -328,52 +328,6 @@ internal class WalletScreenTest {
         onWalletDialogTag().assertDoesNotExist()
     }
 
-    @Test
-    fun `alert is displayed after expiry update failure`() = runTest(dispatcher) {
-        val error = Throwable("oops")
-        val expiredCard = TestFactory.CONSUMER_PAYMENT_DETAILS_CARD.copy(expiryYear = 1999)
-        val linkAccountManager = FakeLinkAccountManager()
-        linkAccountManager.listPaymentDetailsResult = Result.success(
-            ConsumerPaymentDetails(paymentDetails = listOf(expiredCard))
-        )
-        linkAccountManager.updatePaymentDetailsResult = Result.failure(error)
-        val viewModel = createViewModel(linkAccountManager)
-        composeTestRule.setContent {
-            DefaultLinkTheme {
-                WalletScreen(
-                    viewModel = viewModel,
-                    showBottomSheetContent = {},
-                    hideBottomSheetContent = {}
-                )
-            }
-        }
-        composeTestRule.waitForIdle()
-
-        onWalletPayButton().assertIsNotEnabled()
-
-        viewModel.expiryDateController.onRawValueChange("1225")
-        viewModel.cvcController.onRawValueChange("123")
-
-        composeTestRule.waitForIdle()
-
-        onWalletPayButton()
-            .assertIsEnabled()
-            .performClick()
-
-        composeTestRule.waitForIdle()
-
-        onWalletDialogTag()
```

Comment:
The dialog shows up but doesn't seem to be rendered on the compose tree 😢 

<img width="466" alt="image" src="https://github.com/user-attachments/assets/2155072a-e477-44e6-aec1-c106755a0a6f" />